.github/workflows/build_deb.yml

```diff
@@ -51,20 +55,13 @@ jobs:
       with:
         name: ${{ inputs.tar-file-name }}
         path: build_src
-    - name: Register the kolibri PPA
-      id: ppa
-      run: |
-        sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys AD405B4A
-        sudo add-apt-repository -y -u -s ppa:learningequality/kolibri-proposed
-    - name: Install Ubuntu dependencies
-      run: sudo apt install -y devscripts debhelper dh-python python3-all python3-pytest po-debconf
-    - name: Download the most recent kolibri from the PPA
-      run: apt-get -y source kolibri
+    - name: Install Debian build package dependencies
+      run: sudo apt install -y devscripts debhelper dh-python python3-all python3-pytest po-debconf python3-setuptools
     - name: Run the build
       run: make kolibri.deb
     - name: Get DEB filename
       id: get-deb-filename
-      run: echo "::set-output name=deb-file-name::$(ls dist | grep .deb | cat)"
+      run: echo "::set-output name=deb-file-name::$(ls dist | grep all.deb | cat)"
```

Comment:
Is the output filename now always called `all.deb` or am I misunderstanding?

If the former, we should add a step in the `make kolibri.deb` command that names it with the same naming format we currently use.

Reply:
The output of `dpkg-buildpackage` generates several archives, including a `*.deb` file. Keeping the _.deb_ filter was capturing files with the word _debian_ in its name, this _all.deb_ filter captures only the binary package (in Debian, packages running in all the OS as those for Python always end in _all.deb_ to state they don't depend on the microprocessor architecture as others ending in _amd64.deb_, _i386.deb_ , etc. 

However, to keep it as similar as it was before and remove this change, I'm adding a `-A `flag to the build command so the only file containing deb in its name is the one with deb in its filename.

Reply:
Ah, ok - so there are now several `.deb` files, but this will filter to the `-all.deb` or something similar? Which does still have the Kolibri version in the name, got it!

Reply:
The output of building a deb produces these files (only one deb in our case but in other softwares it could be more than one):
- kolibri-source_0.15.12-0ubuntu1.dsc
- kolibri-source_0.15.12-0ubuntu1_amd64.changes
- kolibri_0.15.12-0ubuntu1_all.**deb**
- kolibri-source_0.15.12-0ubuntu1.**deb**ian.tar.gz
- kolibri-source_0.15.12-0ubuntu1_amd64.buildinfo
- kolibri-source_0.15.12.orig.tar.gz

After adding the -A flag, there are less, so the `-all.deb` filter is not needed and we can keep the previous `deb` filter.
- kolibri-source_0.15.12-0ubuntu1_all.buildinfo
- kolibri-source_0.15.12-0ubuntu1_all.changes
- kolibri-source_0.15.12.orig.tar.gz
- kolibri_0.15.12-0ubuntu1_all.**deb**

Reply:
Gotcha, thanks for the detailed explanation!